apps/bfd-pipeline/bfd-pipeline-app/src/main/java/gov/cms/bfd/pipeline/app/PipelineManager.java

```diff
@@ -271,6 +271,8 @@ private void handleJobFailure(PipelineJobRecordId jobRecordId, Exception excepti
         jobRecordStore.recordJobFailure(jobRecordId, new PipelineJobFailure(exception));
         jobsEnqueuedHandles.remove(jobRecordId);
       }
+      LOGGER.error(
+          "Handle job failure in Pipeline: " + exception.getMessage() + " and " + exception);
```

Comment:
can't you just do:
 LOGGER.error("Handle job failure in Pipeline: " + exception.getMessage(), e);

Reply:
done